CMakeLists.txt

```diff
@@ -388,9 +390,18 @@ else()
 
 endif()
 
+if(ENABLE_TBB_DEVICE)
+  # Uses  unofficial FindTBB module (https://github.com/justusc/FindTBB) in ${CMAKE_SOURCE_DIR}/cmake"
+  find_package(TBB REQUIRED)
+  add_library(tbb_library INTERFACE)
+  target_include_directories(tbb_library INTERFACE ${TBB_INCLUDE_DIRS})
+  target_compile_definitions(tbb_library INTERFACE ${TBB_DEFINITIONS})
+  target_link_libraries(tbb_library INTERFACE ${TBB_LIBRARIES})
 
-
-
+  message(STATUS "TBB_INCLUDE_DIRS ${TBB_INCLUDE_DIRS}")
```

Comment:
is this debug, or should it stay? is this in line with other pocl cmake output?

Reply:
this is in line with other pocl cmake output